web/src/main/java/org/springframework/security/web/DefaultSecurityFilterChain.java

```diff
@@ -48,7 +49,18 @@ public DefaultSecurityFilterChain(RequestMatcher requestMatcher, Filter... filte
 	}
 
 	public DefaultSecurityFilterChain(RequestMatcher requestMatcher, List<Filter> filters) {
-		logger.info(LogMessage.format("Will secure %s with %s", requestMatcher, filters));
+		if (requestMatcher instanceof IgnoreRequestMatcher) {
```

Comment:
Maybe we could start with something simpler like this:

```java
if (filters.isEmpty()) {
    logger.info(LogMessage.format("Will not secure %s", requestMatcher));
} else {
    logger.info(LogMessage.format("Will secure %s with %s", requestMatcher, filters));
}
```
If Spring Security has no filters assigned to a given request matcher, then it's true that Spring Security will not be securing that request.